app/models/manageiq/providers/vmware/infra_manager.rb

```diff
@@ -75,11 +75,8 @@ def validate_remote_console_webmks_support
     end
 
     def self.use_vim_broker?
-      return false unless MiqVimBrokerWorker.has_required_role?
-      cfg = ::Settings.webservices.use_vim_broker
-      return true if cfg == "force"
-      return true if cfg && Sys::Platform::OS == :unix
-      false
+      return false if defined?(Rails::Console) && (!::Settings.development || !::Settings.development.use_vim_broker_in_console)
```

Comment:
You should not need the check for `::Settings.development`, as it will always exist by nature of the key being in the core template.

Reply:
@Fryguy having this here means it is not merged lockstep with another repo.

But I can remove. It just adds to the orchestration effort